### DISCUSSION TITLE
Bento grid

### DISCUSSION THREAD

**Discussion by MuhammadKaifNazeer** [shadcn-ui/ui#3896] [Blocks Request]
### Description

bento grid
![clt45wpv90000y6nxwkjy2fzd](https://github.com/shadcn-ui/ui/assets/122388541/89042965-2725-4962-ae0c-59919ef9865d)

**Comment by biapar** [comment]
Hi,
Have you the code?

**Reply by MuhammadKaifNazeer** [reply]
I find this on the Supabase's website, but can make it

**Comment by biapar** [comment]
This is the mine...

![CleanShot 2024-08-26 at 16 34 07@2x](https://github.com/user-attachments/assets/9974208c-7c1d-47c0-95cc-999bf3bfd0af)
